resources/user-agents/browsers/firefox/firefox-4-2.json

```diff
@@ -3,7 +3,7 @@
   "versions": ["4.2"],
   "sortIndex": 3163,
   "lite": false,
-  "standard": true,
+  "standard": false,
```

Comment:
I removed this from Standard since it's an Alpha release.

It doesn't show up on this table: https://en.wikipedia.org/wiki/Firefox_version_history#Release_history

But I did find some discussion about people downloading it from the nightlies.  5.0 was the next official release after 4.0.